lib/enum/arrays.rb

```diff
@@ -1,24 +1,39 @@
+require 'pry-byebug'
+
 class Array
 
-  def sum_of_numeric_elements
+  def initialize (elements)
+    @elements = e
+  end
+
+  def sum_of_numeric_elements #using e for elements.
+    select { [e] e.is_a? Numeric }.reduce(:+) #is_a? to return a boolean value, that if e is a numeric element then add them and return the sum.
```

Comment:
I'll assume for the rest of this assignment that you were writing `|e|`, using the correct syntax with pipes. It's also best practice to use intuitive names for your local variables. In this case, that might be `|element|` or `|item|`.